canonical-web-and-design/vanilla-framework
# Fix side navigation overlapping with global nav

## Done

Fixes #3075

## QA

- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/ or [demo]
- Check documentation layout example on small screen
- Make sure side navigation drawer is not covered by global navigation


## Screenshots

<img width="650" alt="Screenshot 2020-05-20 at 08 54 23" src="https://user-images.githubusercontent.com/83575/82414566-84eafd80-9a77-11ea-933b-8aca9d6bad38.png">
